bundles/org.openhab.persistence.influxdb/src/main/resources/OH-INF/config/config.xml

```diff
@@ -81,6 +81,14 @@
 			<default>false</default>
 		</parameter>
 
+		<parameter name="useMetadataMeasurementName" type="boolean" required="true" groupName="misc">
```

Comment:
Add that parameter also to the documentation in the README. 
And a reference where it says ` with names equal to the...`
Also, it's important to document in the README the name of the metadata key the user must use.

Reply:
Will add this and all other Metadata configuration to README as a complete new section.

> And a reference where it says with names equal to the...

I don't understand what you want me to do? Can you please explain?

Reply:
In line 9 of the README:
Change:

    - The states of an item are persisted in _measurements_ points with names equal to the name of the item, or the alias, if one is provided. In both variants, a _tag_ named "item" is added, containing the item name.

for
  - The states of an item are persisted in _measurements_ points with names equal to the name of the item, its alias, or from some metadata depending on the configuration. In all variants, a _tag_ named "item" is added, containing the item name.

Reply:
Changed